clean

```diff
@@ -20,7 +20,7 @@ endif
 ( cd inc ; /bin/rm -f *.inc namelist.default )
 
 
-set echo
+#set echo
```

Comment:
@davegill 
Can we just remove this line altogether, instead of commenting it out?

Reply:
Done